apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-full-view-presenter.tsx

```diff
@@ -14,7 +15,7 @@ export interface ResourceDetailFullViewPresenterProps {
     // TODO: Refactor the name property to eliminate this conditional type
     name: IMultilingualText | string;
     type: ResourceType;
-    children: ReactNode;
+    children?: ReactNode;
```

Comment:
I assume this would be ok?

Reply:
Yea. I don't see any risks as long as a children prop isn't required to render this generic presenter component.

Reply:
children aren't required for just showing the name, id and resource type, which in some cases may be all that's needed.